test/legacy-component-aliases.test.js

```diff
@@ -0,0 +1,151 @@
+ * - https://github.com/vaadin/web-components/issues/1993
+ *
+ * TODO: Remove after V23
+ */
+import { expect } from '@esm-bundle/chai';
+
+import { AccordionElement } from '@vaadin/vaadin-accordion';
+import { AccordionPanelElement } from '@vaadin/vaadin-accordion/vaadin-accordion-panel';
+import { AppLayoutElement } from '@vaadin/vaadin-app-layout';
+import { DrawerToggleElement } from '@vaadin/vaadin-app-layout/vaadin-drawer-toggle';
+import { AvatarGroupElement } from '@vaadin/vaadin-avatar';
+import { AvatarElement } from '@vaadin/vaadin-avatar/vaadin-avatar';
+import { BoardElement } from '@vaadin/vaadin-board';
+import { BoardRowElement } from '@vaadin/vaadin-board/vaadin-board-row';
+import { ButtonElement } from '@vaadin/vaadin-button';
+import { ChartElement } from '@vaadin/vaadin-charts';
+import { CheckboxElement } from '@vaadin/vaadin-checkbox';
+import { CheckboxGroupElement } from '@vaadin/vaadin-checkbox/vaadin-checkbox-group';
+import { ComboBoxElement } from '@vaadin/vaadin-combo-box';
+import { ComboBoxLightElement } from '@vaadin/vaadin-combo-box/vaadin-combo-box-light';
+import { ConfirmDialogElement } from '@vaadin/vaadin-confirm-dialog';
+import { ContextMenuElement } from '@vaadin/vaadin-context-menu';
+import { CookieConsentElement } from '@vaadin/vaadin-cookie-consent';
+import { CrudElement } from '@vaadin/vaadin-crud';
+import { CustomFieldElement } from '@vaadin/vaadin-custom-field';
+import { DatePickerElement } from '@vaadin/vaadin-date-picker';
+import { DateTimePickerElement } from '@vaadin/vaadin-date-time-picker';
+import { DialogElement } from '@vaadin/vaadin-dialog';
+import { FormLayoutElement } from '@vaadin/vaadin-form-layout';
+import { FormItemElement } from '@vaadin/vaadin-form-layout/vaadin-form-item';
+import { GridElement } from '@vaadin/vaadin-grid';
+import { GridColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-column';
+import { GridColumnGroupElement } from '@vaadin/vaadin-grid/vaadin-grid-column-group';
+import { GridFilterElement } from '@vaadin/vaadin-grid/vaadin-grid-filter';
+import { GridFilterColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-filter-column';
+import { GridSelectionColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-selection-column';
+import { GridSortColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-sort-column';
+import { GridSorterElement } from '@vaadin/vaadin-grid/vaadin-grid-sorter';
+import { GridTreeColumnElement } from '@vaadin/vaadin-grid/vaadin-grid-tree-column';
+import { GridTreeToggleElement } from '@vaadin/vaadin-grid/vaadin-grid-tree-toggle';
+import { GridProElement } from '@vaadin/vaadin-grid-pro';
+import { GridProEditColumnElement } from '@vaadin/vaadin-grid-pro/vaadin-grid-pro-edit-column';
+import { IconElement } from '@vaadin/vaadin-icon';
+import { IconsetElement } from '@vaadin/vaadin-icon/vaadin-iconset';
+import { ItemElement } from '@vaadin/vaadin-item';
+import { ListBoxElement } from '@vaadin/vaadin-list-box';
```

Comment:
Looks like there is an issue with importing `ListBoxElement` specifically, when running the tests in CI. Checking.

Reply:
Fixed after rebase.